src/cript/api/api.py

```diff
@@ -81,6 +81,12 @@ class API:
     _internal_s3_client: Any = None  # type: ignore
     # trunk-ignore-end(cspell)
 
+    # Advanced User Tip: Disabling Node Validation
```

Comment:
Can we make sure this shows up in the documentation of this attribute?

Reply:
as far as I know MkDocs will hide all private attributes, but I think we can put it in the class `attributes` docstrings and it will show up in the documentation. Am I understanding you correctly?

Reply:
yes, let's put it there.
It is not a private attribute though (no `_`).

Do you know how the format looks like for that?

Reply:
yep!

this is not the PR that I did it on, but this is the docstrings format for it
https://github.com/C-Accel-CRIPT/Python-SDK/blob/8e50ca1f87a8ab9b2291d13de86ce77142955d01/src/cript/api/api.py#L54-L75

I can also help as well if you bump into issues as I have a lot of experience with documentation

Reply:
if it is not a private variable and not needed for internal, then we can probably rename/refactor that part as well, I think that would be even cleaner, but it would probably need more work and probably a different PR.